packages/search/src/search.stories.jsx

```diff
@@ -10,10 +10,9 @@ export default { title: 'Search', component: Search };
 const BoxedSearch = ( props ) => (
 	<div style={ { position: 'relative', width: '270px', height: '50px' } }>
 		<Search
-			__={ ( str ) => str }
 			placeholder="Search..."
 			fitsContainer
-			onSearch={ ( search ) => console.log( 'Searched: ', search ) } // eslint-disable-line no-console
+			onSearch={ ( search ) => console.log( 'Searched:', search ) } // eslint-disable-line no-console
```

Comment:
Nit: was the removal of the space after `Searched:` intentional?

Reply:
Yes, apparently `console.log` will add a space between each string argument:

```js
console.log('hello', 'world') // => "hello world"
```